bin/convertToIcns

```diff
@@ -48,23 +47,12 @@ trap cleanUp EXIT
 mkdir -p "${TEMP_DIR}"
 mkdir -p "${ICONSET}"
 
-PNG_PATH="${TEMP_DIR}/icon.png"
-${CONVERT_TO_PNG} "${SOURCE}" "${PNG_PATH}"
+sips -s format tiff "$SOURCE" --out "$TEMP_DIR/icon_128.tiff" -z 128 128 2>&1 > /dev/null & 
```

Comment:
Although on a similar note, I just confirmed that since `tiff2icns`’s limit is `512`, it makes smaller `icns` files than [the `sips`-only solution](https://github.com/jiahaog/nativefier/issues/199#issue-152902368), which means `tiff2icns` does not take into account the `@2x` screens, which are only getting more common.